packages/terra-application-links/src/tabs/ApplicationTabs.jsx

```diff
@@ -176,19 +178,26 @@ class ApplicationTabs extends React.Component {
         collapsedTabs.push(<CollapsedTab {...tabProps} />);
       }
     });
+    const theme = this.context;
+    const tabClass = classNames(cx(
+      'tabs-container',
+      { 'is-calculating': this.isCalculating },
+      alignment,
+      theme.className,
+    ));
```

Comment:
This class list is being applied to an inner div tabs wrapper, adding the theme class name here probably isn't necessary. 

Wrapping `classNames` and `cx` together is used when applying customProps.classNames to the class list without binding it to the stylesheet. If an unbound class isn't being used you don't need to wrap them together, Only `cx` is necessary.

```js
classNames([
  cx('bound-class'), 
  customProps.className // Add the custom class to the classList without binding it to the stylesheet
])
```